addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -423,11 +471,31 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
      * ie: The Job you apply for if the form is on that job's page.
      */
     addActionField: function (previewMode, value, params) {
+        // Remove old property fields.
+        authorizedFieldsCache.get(this.$target[0], this._rpc.bind(this)).then((fields) => {
```

Comment:
> Didn't you tell me this should be awaited ? :thinking:

Then `.then(...)` serves that purpose, so that the fields are available before performing the cleanup.

Crash is under investigation.

Reply:
Marked this as resolved since I couldn't reproduce the issue described here... cc @sobo-odoo if you still manage to reproduce it, just let me know :+1:

Reply:
I didn't retest but it is supposed to be fixed, given the answer [here] (on the last comment of my review) :slightly_smiling_face:

[here]: https://github.com/odoo/odoo/pull/119834#discussion_r1193833276